.circleci/config.yml

```diff
@@ -592,7 +592,12 @@ jobs:
       - run: *run_build
       - store_artifacts: *artifacts_solc
       - store_artifacts: *artifacts_tools
-      - persist_to_workspace: *artifacts_build_dir
+      - persist_to_workspace:
+          root: .
+          paths:
+            - build/solc/solc
+            - build/test/soltest
+            - build/test/tools/solfuzzer
```

Comment:
This is the main workaround. osx moves these files in a subdirectory ``build``, while the rest has one directory level less.